.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "BinarySearch.C": "cpp"
+    }
+}
```

Comment:
Remove this

Reply:
Sir, I am unable to remove this.would you help me please?

Reply:
Just remove it from the staging area and make a new commit